### ISSUE TITLE
Getting an error when setting up the installation

### ISSUE THREAD

**Issue by IonicaBizau** [zsh-users/zsh-syntax-highlighting#313]
`echo "source ${(q-)PWD}/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh" >> ~/.zshrc` ends with:

```sh
-bash: source ${(q-)PWD}/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh: bad substitution
```

**Comment by danielshahaf** [comment]
`${(q-)foo}` is a zsh-specific syntax that may not work in bash.

I assume you've already figured out how to make it work (assuming the local git clone's directory name's absolute path is shell-safe, just dropping the `(q-)` is all that's needed), so what is this bug report about?  Are you asking us to make the installation instructions work in `bash` too?  (That might be a good idea; I just want to be sure I understand which question you're asking before answering it.)

Nice ASCII art on your profile page :-)

**Comment by IonicaBizau** [comment]
I'm not that familiar with `zsh`, but I want to use it. That's interesting to know. Thanks!

> Nice ASCII art on your profile page :-)

Generated with [this tool](https://github.com/IonicaBizau/github-contributions). 😄

**Comment by danielshahaf** [comment]
The reference documentation is [in zshexpn(1)](http://zsh.sourceforge.net/Doc/Release/Expansion.html#Parameter-Expansion-Flags).  I recommend #zsh on freenode (or the zsh-users@ list) for general zsh questions.

I've also committed a few minor fixes (162c826, a00ed24) to INSTALL.md.  These don't actually mention the need to run the example code from `zsh` since I haven't figured out a non-cumbersome way to state that information.

Thanks for the pointer!
